lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -668,11 +667,10 @@ instance LiftHandler ErrSignTx where
 instance LiftHandler ErrDecodeExternalTx where
     handler = \case
         ErrDecodeExternalTxWrongPayload err ->
-            apiError err404 MalformedTxPayload $ mconcat
-                [ "I couldn't decode the payload that seems to be an "
-                , "externally-signed transaction due to: ", pretty err, ". "
-                , "Make sure to send a base64-encoded binary blob, in the "
-                , "proper binary format of the already-serialized transaction."
+            apiError err400 MalformedTxPayload $ mconcat
+                [ "I couldn't verify that the payload has the correct binary "
+                , "format due to: ", pretty err, ". I won't send it to the node"
+                , ". Make sure the blob has the correct binary format."
```

Comment:
Recommend changing "blob" to "payload" for consistency with earlier usage.

Reply:
done